lisa/transformers/azure_hyperv.py

```diff
@@ -0,0 +1,64 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.node import quick_connect
+from lisa.tools import PowerShell
+from lisa.transformer import Transformer
+from lisa.util import field_metadata
+
+
+@dataclass_json
+@dataclass
+class AzureHypervTransformerSchema(schema.Transformer):
+    # SSH connection information to the node
+    connection: Optional[schema.RemoteNode] = field(
+        default=None, metadata=field_metadata(required=True)
+    )
+
+
+class AzureHypervTransformer(Transformer):
```

Comment:
The name is not clear enough on its purpose. How about `HyperVPreparationTransformer`?

Reply:
Renamed the transformer to "hyperv_preparation"